README.md

```diff
@@ -8,21 +8,188 @@
+     --header 'content-type: application/x-www-form-urlencoded' ^
+     --data 'grant_type=authorization_code&code=<CODE>&redirect_uri=<YOUR_REDIRECT_URI>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+     ```
+
+   This command will return the access token necessary for API calls.
+
+   ```json
+   {
+     "token_type": "bearer",
+     "refresh_token": "<Refresh Token>",
+     "access_token": "<Access Token>",
+     "expires_in": 1800
+   }
+   ```
+
+5. Store the access token securely for use in your application.
+
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `HubSpot Marketing Emails` connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `hubspot.marketing.emails` module and `oauth2` module.
+
+```ballerina
+import ballerinax/hubspot.marketing.emails as hsmemails;
+import ballerina/oauth2;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `Config.toml` file and, configure the obtained credentials in the above steps as follows:
+
+   ```toml
+    clientId = <Client Id>
+    clientSecret = <Client Secret>
+    refreshToken = <Refresh Token>
+   ```
+
+2. Instantiate a `hsmevents:ConnectionConfig` with the obtained credentials and initialize the connector with it.
+
+    ```ballerina 
+    configurable string clientId = ?;
+    configurable string clientSecret = ?;
+    configurable string refreshToken = ?;
+
+    final hsmemails:ConnectionConfig hsmeventsConfig = {
+        auth : {
+            clientId,
+            clientSecret,
+            refreshToken,
+            credentialBearer: oauth2:POST_BODY_BEARER
+        }
+    };
+
+    final hsmemails:Client hsmevents = check new (hsmeventsConfig, "https://api.hubapi.com/marketing/v3/emails");
+    ```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations. A sample usecase is shown below.
+
+#### Create a Marketing Event
+    
+```ballerina
+public function main() returns error? {
+    hsmemails:AggregateEmailStatistics emailStatistics = check hubspotClient->/marketing/v3/emails/statistics/list({}, 
+     {
+         startTimestamp: "2024-12-12T04:27:02Z",
+         endTimestamp: "2024-12-19T04:27:02Z"
+     });
+}
+```
 
 ## Examples
 
-The `HubSpot Marketing Emails` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-hubspot.marketing.emails/tree/main/examples/), covering the following use cases:
+The `Hubspot Marketing Emails` connector provides practical examples illustrating usage in various scenarios. Explore these examples, covering use cases:
 
-[//]: # (TODO: Add examples)
+1. [Bulk Change Reply To Email](./BulkChangeReplyEmail/) - Change the Reply To and Custom Reply To email address of all draft emails
```

Comment:
Check whether these paths are correct. It should use the relative path from the file they are mentioned.

```suggestion
1. [Bulk Change Reply To Email](./examples/BulkChangeReplyEmail/) - Change the Reply To and Custom Reply To email address of all draft emails
```